agent/consul/internal_endpoint_test.go

```diff
@@ -175,6 +213,21 @@ func TestInternal_NodeDump(t *testing.T) {
 	if !foundFoo || !foundBar {
 		t.Fatalf("missing foo or bar")
 	}
+
+	require.Equal(t, 1, len(out2.ImportedDump))
+	require.Equal(t, "peer1", out2.ImportedDump[0].PeerName)
+	require.Equal(t, "foo-peer", out2.ImportedDump[0].Node)
+
+	// test filter on nodes
```

Comment:
There's a test for filtering below: `TestInternal_NodeDump_Filter`, this should be moved over thtere